lib/pages/projects/project_widget.dart

```diff
@@ -90,13 +95,12 @@ class ProjectRow extends StatelessWidget {
         project.name,
         key: ValueKey("${project.name}_${project.id}"),
       ),
-      trailing: Container(
-        height: 10.0,
-        width: 10.0,
-        child: CircleAvatar(
-          key: ValueKey("dot_${project.name}_${project.id}"),
-          backgroundColor: Color(project.colorValue),
-        ),
+      trailing: IconButton(
+        icon: Icon(Icons.delete),
+        onPressed: () {
+          projectDb.deleteProject(project.id);
```

Comment:
We try to not expose DB directly on UI instead do this operation in the bloc.

Reply:
AlertDialog box added please review for any changes